src/Struct/FindologicProduct.php

```diff
@@ -50,6 +53,9 @@ class FindologicProduct extends Struct
     /** @var Context */
     protected $context;
 
+    /** @var SalesChannelContext */
+    protected $salesChannelContext;
+
```

Comment:
`$context` can be replaced with `$salesChannelContext`, instead of added additionally

Reply:
That would also break backwards compatibility. Such a change may be done when we remove this parameter. For now I will add a deprecation notice for this parameter.